modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/MustacheEngineAdapter.java

```diff
@@ -40,7 +41,7 @@ public String getIdentifier() {
     }
 
     public String[] extensions = new String[]{"mustache"};
-    Mustache.Compiler compiler = Mustache.compiler();
+    Mustache.Compiler compiler = Mustache.compiler().withEscaper(Escapers.NONE);
```

Comment:
avoid escaping (quotes in my case for `@SuppressWarnings("unchecked")` but as you can this will also fix generating docs